.eslintrc

```diff
@@ -36,7 +36,15 @@
         "alphabetize": {
           "order": "asc"
         },
-        "groups": ["builtin", "external", "sibling", "parent", "index"]
+        "groups": ["builtin", "external", "sibling", "parent", "index"],
+        "pathGroups": [
+          {
+            "pattern": "*.module.css",
+            "group": "index",
+            "patternOptions": { "matchBase": true },
+            "position": "after"
```

Comment:
Looks for `module.css` and places after the `index` file, which is placed after imports from parents (where subcomponents live)